src/LinAlg/hiopVectorRajaPar.cpp

```diff
@@ -218,6 +221,18 @@ void hiopVectorRajaPar::setToConstant(double c)
     });
 }
 
+void hiopVectorRajaPar::set_to_random_uniform(double minv, double maxv)
+{
+  double* data = data_dev_;
+#ifdef HIOP_USE_GPU
+  hiop::device::array_random_uniform_kernel(n_local_, data);
+  scale(maxv-minv);
+  addConstant(minv);
+#else
+  assert(0&&"not yet");
```

Comment:
Should we just use c++ standard features to set the array to be random here?

Reply:
You are right. My bad. I should change this PR to draft mode since I would like to test hip implementation as well.